sample/CMakeLists.txt

```diff
@@ -1,9 +1,11 @@
+#include files 
 include_directories(${PROJECT_SOURCE_DIR}/commandlineparser/include)
```

Comment:
Try not including this.. I thing target_link_libraries(SampleApp cmd_lib) should automatically do this.. but I am not sure please check